### DISCUSSION TITLE
Category field: multiple category source field – like the entry field

### DISCUSSION THREAD

**Discussion by wedot** [craftcms/cms#9019] [Ideas]
### Description

In the entry field I can select entries from multiple entry sections, it would be nice to be able to do the same with the  category field. 

So that I can handle the output by my self via twig.

**Comment by davidhellmann** [comment]
I remember there are no more categories in 4.0 and all is under a Content merged together? We use all the time a channel for categories cause there are Multi Site Benefits.

**Comment by iuscare** [comment]
^ Also interested if this is still valid, since the feature request is added as a 4.0 milestone and I thought categories will get replaced.

**Comment by brandonkelly** [comment]
Yeah that’s still the plan. I added the 4.0 milestone because the issue will be resolved 4.0, in a Milton-esque “we fitched the glitch” sort of way :)

**Comment by wmdhosting** [comment]
Is this solved in 4,   now i use plugin for this.. but seem will not be ported to 4..

**Reply by brandonkelly** [reply]
Sorry not yet.
